README.md

```diff
@@ -73,7 +69,7 @@ Events are created every execution of an example - currently no check is done if
 All examples use a common set of arguments:
 
 ```bash
-export AUTH="-u $TEST_ARCHIVIST -t $TEST_AUTHTOKEN_FILENAME $TEST_VERBOSE $TEST_PROOF_MECHANISM"
```

Comment:
The problem with removing altogether is that in about 2 months we'll have the Khipu replacement and will have to put it all back again. Especially given that testing the new stuff is important I think we should leave this